### ISSUE TITLE
v1.7.4 - App unresponsive / crashes

### ISSUE THREAD

**Issue by sunjaxx** [blackholeearth/Win10_BrightnessSlider#114]
Once every couple of days the app becomes unresponsive and crashes. 

Not sure this is related, but is the only thing i see in the log:

```
17/03/2022 08:47:47 ==>  Caller(File,Member,Line): wmiMonFn.cs, GetBrightness, 85 
System.Management.ManagementException: Not supported 
   at System.Management.ManagementException.ThrowWithExtendedInfo(ManagementStatus errorCode)
   at System.Management.ManagementObjectCollection.ManagementObjectEnumerator.MoveNext()
   at Win10_BrightnessSlider.wmiMonFn.GetBrightness(String wmi_InstanceName) 
```

Any suggestion?

**Comment by blackholeearth** [comment]
is this datetime really close to unresponsiveness and crashes

**Comment by sunjaxx** [comment]
Yes this was just few seconds after a crash.
I keep the pc on and usually put it to sleep (not off) at night. I restart it maybe every week.

**Comment by blackholeearth** [comment]
released try it 1.7.15

**Comment by sunjaxx** [comment]
I just installed 1.7.15: I will report back! thanks
